src/stores/configuration.js

```diff
@@ -0,0 +1,12 @@
+import { defineStore } from 'pinia'
+
+export const useConfigurationStore = defineStore('configuration', {
+  state () {
+    return {
+      isConfigLoaded: false,
+      observationPortalUrl: '',
+      rtiBridgeUrl: '',
+      thumbnailArchiveUrl: ''
+    }
```

Comment:
the aforementioned telescopeStatusApiUrl could live here as a global config item